c2corg_ui/static/js/utils.js

```diff
@@ -2,6 +2,26 @@ goog.provide('app.utils');
 
 
 /**
+ * @param {string} type Short document type code.
+ * @return {string} Full document type name.
+ */
+app.utils.getDoctype = function(type) {
+  // TODO add other types
+  switch (type) {
+    case 'w':
+      return 'waypoints';
+    case 'r':
+      return 'routes';
+    case 'o':
+      return 'outings';
+    case 'i':
+      return 'images';
+    default:
+      return '';
```

Comment:
`goog.asserts` is already used elsewhere (e.g. [here](https://github.com/c2corg/v6_ui/blob/e6a9a14398131a6e00993968d1c121d2088ffa93/c2corg_ui/static/js/map/diffmap.js#L161)), so you don't have to change the build config. But it is `goog.asserts` and not `goog.assert` (note the "s").